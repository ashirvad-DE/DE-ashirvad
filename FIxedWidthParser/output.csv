John,25M,Eng,ineer
Jane,30F,Doc,tor
