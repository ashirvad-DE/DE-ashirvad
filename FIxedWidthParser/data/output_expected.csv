John,25,M,Engineer
Jane,30,F,Doctor
